product/distribution/src/repository/conf/template-manager/domain-template/SensorAnomalyDetectionDomain.xml

```diff
@@ -0,0 +1,394 @@
+<!--
+  ~ Copyright (c)  2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<domain name="SensorAnomalyDetectionDomain">
+    <description>Domain for sensor anomaly detection</description>
+    <scenarios>
+        <scenario type="LinearRegressionWithLengthWindow">
+            <description>Configure a sensor anomaly detection scenario to receive alerts for a given stream of your
```

Comment:
Anomalies in sensor reading will be detected using liner regression over a defined number of events and those anomalies are displayed in the alert gadget at alert dashboard.